core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -657,6 +657,10 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    *
    * Load data from a flat binary file, assuming the length of each record is constant.
    *
+   * '''Note:''' Normally getBytes returns an array padded with extra values,
```

Comment:
Does the end user ever need to call getBytes() given that this RDD is of type Array[Byte]? Does the user even need to know that this feature uses FixedLengthBinaryInputFormat underneath. And if they dont, then this note can be simplified and mentioned in terms of the RDD, something like "Each record in this DStream/RDD is of fixed length with padded bytes / right length / etc."